quicktests/overlaying/overlaying.js

```diff
@@ -229,12 +229,12 @@ function loadAllQuickTests(quicktestsPaths, firstQTBranch, secondQTBranch){
       var className = "quicktest " + name;
       var div = d3.select("#results").append("div").attr("class", className);
       div.insert("label").text(name);
-      var firstsvg = div.append("div").attr("class", "first").append("svg").attr({width: svgWidth, height: svgHeight});
-      var secondsvg = div.append("div").attr("class", "second").append("svg").attr({width: svgWidth, height: svgHeight});
+      var firstdiv = div.append("div").attr("class", "first").append("div").attr({width: svgWidth, height: svgHeight});
```

Comment:
I don't think you should use the "div" name to identify elements; instead use "element". the important change you're making here is a switch from `SVGElement` to `HTMLElement`; the tag itself is not so important (it could conceivably be a span, for instance).

this applies to all the quicktests